src/schema/me/followed_shows.ts

```diff
@@ -40,6 +40,11 @@ const FollowedShows: GraphQLFieldConfig<void, ResolverContext> = {
   args: pageable({
     status: EventStatus,
     sort: PartnerShowSorts,
+    dayThreshold: {
+      type: GraphQLString,
```

Comment:
This should be a `GraphQLInt`?

Reply:
yes to ☝️

Reply:
I can update this @orta

Reply:
He already did here: https://github.com/artsy/metaphysics/pull/1584

Reply:
nvm looks to be already `int` on `master` :)